pkg/validation/validate.go

```diff
@@ -11,7 +11,9 @@ import (
 )
 
 const (
-	ReasonLabel = "reason"
+	ReasonLabel    = "reason"
+	MissingStreams = "error at least one valid stream is required for ingestion"
```

Comment:
nit: I would use the same naming convention as for other error messages

```suggestion
	MissingStreamsErrorMsg = "error at least one valid stream is required for ingestion"
```